packages/eslint-plugin/lib/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @fileoverview Agoric-specific plugin
+ * @author Agoric
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var requireIndex = require("requireindex");
```

Comment:
I suppose that it doesn’t matter that this is so tightly coupled to Node.js. 🤷‍♂️

Reply:
Yup.